conduct.md

```diff
@@ -120,15 +120,15 @@ TODO: Write an introductory paragraph.
 
 #### Acceptable Conduct
 
-- TODO
-- TODO
-- TODO
+- Openness. It is important to be vulnerable in your group so that they understand your strengths and weaknesses within coding. Being open will help your team members know what you are capable of doing so they do not assign you something that you would not understand. This could prevent confusion and reserve time. Being open can also allow your team members to help you with problems with your code that you did not catch or understand.
+- Retaining team members ideas. This is a form of respecting everyone's thoughts and opinions. This means to keep or reserve someones idea to let your teammates know that they are heard within the group and their opinions matter.
+- Humbleness. Being humble is a form of humility it is important for everyone in a team to understand that it is not about one person but that it is about the team.
 
 #### Unacceptable Conduct
 
-- TODO
-- TODO
-- TODO
+- Resentment. To resent means to feel anger or bitterness towards someone. This will disrupt the flow of the team and can lead to miscommunication and/or using inappropriate language. This a selfish act and is not respecting the group. It causes tension increases conflicts.
+- Lack of Communication. Within a group, communication is key to keep the team on track. One way to practice good communication is by giving good criticism which is a form of respect.
```

Comment:
I'm not entirely sure what you mean by this:
"One way to practice good communication is by giving good criticism which is a form of respect.". What is "good" criticism and how is it a form of respect?

Another note is, again, this point is more so concerning the communication section. In my opinion, it would be sufficient to just say this:
```suggestion
- Lack of Communication and miscommunication are primary causes of conflict. Abiding by the conducts described in the communication section prevents escalation and avoids issues from the start. 
```
However, I leave that up to you and the team to decide.